include/zephyr/drivers/mipi_dbi.h

```diff
@@ -15,7 +15,7 @@
  *     1. 9 write clocks per byte, final bit is command/data selection bit
  *     2. Same as above, but 16 write clocks per byte
  *     3. 8 write clocks per byte. Command/data selected via GPIO pin
- * The current driver interface only supports type C modes 1 and 3
+ * The current driver interface does not support type C with 16 write clocks (option 2).
```

Comment:
This change should be moved to the prior commit

Reply:
Done.